docker/jenkins/run_integration_tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
-GIT_COMMIT=${GIT_COMMIT:-$(git rev-parse HEAD)}
-cp docker/dockerfiles/bedrock_integration_tests Dockerfile
-docker build -t bedrock_integration_tests:${GIT_COMMIT} --pull=true .
+
+# $1 should be the properties file for this run
+source "$1"
```

Comment:
Thanks! This is the best way I could figure out to inject environment variables via a file. I looked briefly at parsing the properties files as-is in the Groovy and using `withEnv`, but this seemed cleaner, easier, and less error prone.